SayTheirNames/Source/Service/Networking/Requestors/DonationsNetworkRequestor.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  DonationsNetworkRequestor.swift
+//  Say Their Names
+//
+//  Created by evilpenguin on 6/3/20.
+//  Copyright © 2020 Franck-Stephane Ndame Mpouli. All rights reserved.
+//
+
+import Foundation
+
+// MARK: - PetitionUrl
+final class DonationsUrl: BaseNetworkUrl {
+    class var donations: UrlString { return "\(self.base)/api/donations" }
+}
+
+// MARK: - DonationsNetworkRequestor
+typealias DonationsNetworkRequestor = NetworkRequestor
```

Comment:
I honestly think that these type aliases make it more confusing, as one would need to jump one more definition to see what it is. So just an extension on `serverURLString` would be clearer IMO.

Reply:
I would disagree, but I will gladly change it. Changed.